### ISSUE TITLE
simplify ui experience by removing unnessery dialog clicks in standard dialogs for crud

### ISSUE THREAD

**Issue by thomaswoehlke** [Jakarta-EE-Petclinic/petclinic-jakartaee#8]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by ritonglue** [comment]
That's right, if you want to populate the database with many new entities, it's quite annoying to clic back and forth.
